code/src/App.js

```diff
@@ -1,9 +1,50 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+
+import { API_URL } from "./utils/urls"
+import Thought from "./components/Thought"
+import ThoughtForm from "./components/ThoughtForm"
+import Loader from "./components/Loader"
+import LikeCounter from './components/LikeCounter'
 
 export const App = () => {
+  const [thoughts, setThoughts] = useState([])
+  const [newThought, setNewThought] = useState("")
+  const [loading, setLoading] = useState(true)
+  const [likeCounter, setLikeCounter] = useState(0)
+
+  useEffect(() => {
+    fetchThoughts()
+  }, [])
+
+  const fetchThoughts = () => {
+    fetch(API_URL)
+      .then(res => res.json())
+      .then(data => setThoughts(data))
+      .catch(err => (console.log(err)))
```

Comment:
console.log cant be present in the production code.

Reply:
I've cleaned up the code now :)